ios/RCTContacts/RCTContacts.m

```diff
@@ -40,6 +40,38 @@ - (NSDictionary *)constantsToExport
     }];
 }
 
+RCT_EXPORT_METHOD(getContactsMatchingString:(NSString *)string callback:(RCTResponseSenderBlock) callback)
+{
+    CNContactStore *contactStore = [[CNContactStore alloc] init];
+    if (!contactStore)
+        return;
+    [self getContactsFromAddressBook:contactStore matchingString:string callback:callback];
+}
+
+-(void) getContactsFromAddressBook:(CNContactStore *)store
```

Comment:
it hasn't been documented, but in this project I'd prefer to use upto 119 columns on lines. This is the width of github's UI. Though we haven't strictly agreed on this @npomfret @rt2zz . No matter now just thought I'd mention it sooner rather than later.